obsreport/obsreport_exporter.go

```diff
@@ -251,6 +252,10 @@ func endSpan(ctx context.Context, err error, numSent, numFailedToSend int64, sen
 
 func toNumItems(numExportedItems int, err error) (int64, int64) {
 	if err != nil {
+		if e, ok := err.(consumererror.Permanent); ok {
```

Comment:
```suggestion
		var permErr consumererror.Permanent
		if errors.As(err, &permErr) { {
```

Otherwise we ignore wrapped errors, right?

Reply:
Yes, that's a good point. I was making the assumption that we're calling this right after `ConsumeTraces`, but we should assume that the error may be wrapped.